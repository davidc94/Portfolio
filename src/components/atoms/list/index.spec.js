import React from 'react';
import renderer from 'react-test-renderer';
import StyledList from './index';

describe('StyledList component', () => {
  it('render default StyledList as expected', () => {
    const render = renderer.create(
      <StyledList>I have nothing to offer</StyledList>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('render StyledList with a different font size', () => {
    const render = renderer.create(
      <StyledList size="small">EXTRA small</StyledList>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});