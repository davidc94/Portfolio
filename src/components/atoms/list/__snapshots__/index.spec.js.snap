// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledList component render StyledList with a different font size 1`] = `
<li
  className="sc-bdVaJa digvwl"
  size="small"
>
  EXTRA small
</li>
`;

exports[`StyledList component render default StyledList as expected 1`] = `
<li
  className="sc-bdVaJa jhpExi"
>
  I have nothing to offer
</li>
`;
